.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,32 @@
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)